pillarbox-player/docs/MediaSession.md

```diff
@@ -72,7 +73,7 @@ coroutineScope.launch {
 
 [`PillarboxMediaLibraryService`][pillarbox-media-library-service-source] has the same features as
 [`PillarboxMediaSessionService`][pillarbox-media-session-service-source], but it allows the application to provide content with
-[`MediaBrowser`][media-browser-documentation]. More information about [Android auto][android-auto-documentation].
+[`MediaBrowser`][media-browser-documentation]. More information about [Android Auto][android-auto-documentation].
```

Comment:
Not sure which page you wanted to link here for `Android Auto`, but the `[android-auto-documentation]` reference does not exist.

Reply:
Indeed I forgot to copy this link from the other readme.